api/src/lib.rs

```diff
@@ -69,10 +70,21 @@ pub trait Bench {
     fn store(&self) -> &dyn Blockstore;
     /// Looks up a top-level actor state object in the VM.
     /// Returns None if no such actor is found.
-    fn find_actor(&self, id: ActorID) -> anyhow::Result<Option<ActorState>>;
+    fn find_actor(&self, id: ActorID) -> anyhow::Result<Option<Actor>>;
     /// Resolves an address to an actor ID.
     /// Returns None if the address cannot be resolved.
     fn resolve_address(&self, addr: &Address) -> anyhow::Result<Option<ActorID>>;
+
+    /// Get the root cid of the state tree
+    fn state_root(&mut self) -> Cid;
```

Comment:
We don't need both find_actor and state_root. I had initially opted for just a little abstraction by providing the former. If there are good reasons to expose the state root, we can do that instead but should then move the find_actor logic to the wrangler (the version you have commented out there).

Reply:
Ditto for resolve_address.